File: C/1012.c
#include <stdio.h>
 
int main() {
 
double A, B, C;
    scanf("%lf %lf %lf", &A, &B, &C);
    printf("TRIANGULO: %.3f\n", (A * C) / 2 );
    printf("CIRCULO: %.3f\n", 3.14159 * pow(C,2));
    printf("TRAPEZIO: %.3f\n", ((A + B)*C)/2 );
    printf("QUADRADO: %.3f\n", pow(B,2));
    printf("RETANGULO: %.3f\n", A*B);
 
    return 0;
}

File: Javascript/1012.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let values = lines.shift().split(' ');
value = Number(values[0]) * Number(values[2]) / 2;
console.log("TRIANGULO: " + value.toFixed(3));

value = 3.14159 * Math.pow(Number(values[2]),2);
console.log("CIRCULO: " + value.toFixed(3));

value = ((Number(values[0]) + Number(values[1])) * Number(values[2])) / 2;
console.log("TRAPEZIO: " + value.toFixed(3));

value = Math.pow(Number(values[1]),2);
console.log("QUADRADO: " + value.toFixed(3));

value = Number(values[0]) * Number(values[1]);
console.log("RETANGULO: " + value.toFixed(3));